### DISCUSSION TITLE
我的github账户被封禁了，想要使用这个邮箱，无法取消关联邮箱，

### DISCUSSION THREAD

**Discussion by Damon-Suan** [community/community#147325] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

我的github账户被封禁了，想要使用这个邮箱，无法取消关联邮箱。联系支持人员选择手机号没有中国

**Comment by shinybrightstar** [comment]
Hi @Damon-Suan , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
